### ISSUE TITLE
Unable to connect to snap

### ISSUE THREAD

**Issue by marvinmarnold** [mario-christopher/snaps-dynamic-ui#1]
When trying to connect, `endowment:signature-insight` does not exist.

Steps:
- Clone repo
- `yarn`
- `yarn start`
- Click connect
- Accept permissions

<img width="1246" alt="Screenshot 2024-02-26 at 1 12 54 PM" src="https://github.com/mario-christopher/snaps-dynamic-ui/assets/1311030/d0fa1d2e-381e-4c56-b4ea-225386abecb4">

**Comment by marvinmarnold** [comment]
@mario-christopher I'm working on the hackathon and blocked by this btw. thanks!

**Comment by marvinmarnold** [comment]
Got it. Weird, I installed the experimental Metamask in a new Chrome profile yesterday. But now I see the [instructions](https://github.com/MetaMask/snaps/discussions/2138) point to a new install. Now the endowment is working. I'm seeing another issue I'll open separately.
